### ISSUE TITLE
[Bug] SELinux issues for cirrus + fedora 33 + podman

### ISSUE THREAD

**Issue by daniel-wtd** [cirruslabs/cirrus-cli#192]
Hello, thanks for this awesome tool. I am testing a bit after getting this article: https://www.redhat.com/sysadmin/cirrus-cli-podman via reddit.

During my initial tests, everything seems to work fine, except one thing: SELinux is making a bit of a trouble.

### Steps to reproduce:

1. Download cirrus 0.20.1 for linux amd64
2. Create a `.cirrus.yml` like this:
 ```yaml
 container:
   image: "docker.io/library/python:3"

 test_task:
    matrix:
      - name: "lint"
        lint_script:
          - "echo linting"
          - "true"
      - name: "test"
        test_script:
          - "echo testing"
          - "false"
  ```
3. run `cirrus run`

This will lead to the situation, that the image download will be done, but afterwards the output is stale and nothing will happen.

4. Disable SELinux temporarily: `sudo setenforce permissive`
5. run `cirrus run`

This will lead to the a proper result and all tests are executed.

### Visible issues

Via `journalctl -xe`, I was able to identify the SELinux issue:

```
Dez 14 10:26:28 nb01.while-true-do.io audit[10989]: AVC avc:  denied  { write } for  pid=10989 comm="cirrus-ci-agent" name="cli-4403034c-6279-40c9-8a44-cddf2d79523c.sock" dev="tmpfs" ino=5230 scontext=system_u:system_r:container_t:s0:c340,c883 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=sock_file permissive=0
Dez 14 10:26:29 nb01.while-true-do.io audit[10989]: AVC avc:  denied  { write } for  pid=10989 comm="cirrus-ci-agent" name="cli-4403034c-6279-40c9-8a44-cddf2d79523c.sock" dev="tmpfs" ino=5230 scontext=system_u:system_r:container_t:s0:c340,c883 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=sock_file permissive=0
Dez 14 10:26:31 nb01.while-true-do.io audit[10989]: AVC avc:  denied  { write } for  pid=10989 comm="cirrus-ci-agent" name="cli-4403034c-6279-40c9-8a44-cddf2d79523c.sock" dev="tmpfs" ino=5230 scontext=system_u:system_r:container_t:s0:c340,c883 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=sock_file permissive=0
Dez 14 10:26:34 nb01.while-true-do.io audit[10989]: AVC avc:  denied  { write } for  pid=10989 comm="cirrus-ci-agent" name="cli-4403034c-6279-40c9-8a44-cddf2d79523c.sock" dev="tmpfs" ino=5230 scontext=system_u:system_r:container_t:s0:c340,c883 tcontext=unconfined_u:object_r:user_tmp_t:s0 tclass=sock_file permissive=0
```

According to this output, there seems to be an issue using the podman socket.

### additional information

```
$ cirrus --version
cirrus version 0.20.1-5490f13

$ podman --version
podman version 2.2.1

$ rpm -q container-selinux
container-selinux-2.151.0-1.fc33.noarch
```

Since this may also be an issue with podman itself, I will also link/open a podman issue for the same.

EDIT: Link for podman issue https://github.com/containers/podman/issues/8711

**Comment by edigaryev** [comment]
Hello Daniel, thanks for the report!

I'm pretty sure this bug has nothing to do with Podman itself, since this socket is the one that CLI creates for talking with the [cirrus-ci-agent](https://github.com/cirruslabs/cirrus-ci-agent).

I've created a fix in #194 to address this, however, it seems that there's some logic missing on the Podman's side to actually pick up these SELinux settings and I'm currently investigating this.

**Comment by edigaryev** [comment]
This should now be fixed as a part of [0.21.0](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.21.0) release.

**Comment by daniel-wtd** [comment]
I will give it a try :) Thanks a lot.
